docs/tutorial/rust/src/main_game_logic_in_rust.rs

```diff
@@ -20,7 +20,7 @@ fn main() {
     // ANCHOR: game_logic
     // Assign the shuffled Vec to the model property
     let tiles_model = std::rc::Rc::new(sixtyfps::VecModel::from(tiles));
-    main_window.set_memory_tiles(sixtyfps::ModelRc::new(tiles_model.clone()));
+    main_window.set_memory_tiles(sixtyfps::ModelRc::from(tiles_model.clone()));
```

Comment:
Is this also possible?

```suggestion
    main_window.set_memory_tiles(tiles_model.clone().into());
```

Reply:
yes, i'll change that.